.github/ISSUE_TEMPLATE/alpha_feature.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Alpha Feature Requirements
+about: Requirements for promoting a feature to alpha
+---
+
+This page lists the requirements for promoting a feature to alpha. Please check off and document the steps as they are completed.
+
+**Feature Name:** 
+--- 
+
+### Requirements: 
+
+**Design**
+
+- [ ] Requires approved RFC in order to merge. 
+
+**Config**
+
+- [ ] Requires explicit user action to enable (e.g. a config field, config resource, or installation action). 
+
+**Docs**
+
+- [ ] Reference docs are published to istio.io. 
+- [ ] Basic feature docs are published on istio.io describing what the feature does, how to use it, and any caveats. 
+- [ ] Whenever possible, documentation has automated istio.io tests written. 
+- [ ] A reference to the design doc/issue is published on istio.io. 
```

Comment:
Not sure about this requirement, do we actually need this? Do users look at the design docs from the istio.io docs?

Reply:
I'm not sure. I have no idea where I would find this on istio.io. It was in the list in the community repo so I added it here. Happy to remove it.

Reply:
I don't think we link them anywhere today - and probably people will just complain about permissions..

Reply:
I personally think it is too much for users to read, especially our design docs with many options.  people already provided feedback that our istio.io site is too much istio developer focused less  Istio user focused.

Reply:
Agreed and removed. Thanks.